server/routerlicious/packages/services-core/src/cache.ts

```diff
@@ -16,4 +16,10 @@ export interface ICache {
      * Sets a cache value
      */
     set(key: string, value: string, expireAfterSeconds?: number): Promise<void>;
+
+    /**
+     * Deletes a cache value
+     */
+     delete(key: string): Promise<boolean>;
```

Comment:
Let's make this optional? Otherwise if customer implement the `ICache` interface, this will break their implementations, and this would also break all implementations in FRS as well.

Reply:
Updated so delete is optional